src/config.rs

```diff
@@ -266,3 +272,57 @@ impl SizeTieredCompactionSchedulerOptions {
         }
     }
 }
+
+/// Garbage collector options.
+#[derive(Clone)]
+pub struct GarbageCollectorOptions {
+    /// Garbage collection options for the manifest directory.
+    pub manifest_options: Option<GarbageCollecterDirectoryOptions>,
+
+    /// Garbage collection options for the WAL directory.
+    pub wal_options: Option<GarbageCollecterDirectoryOptions>,
+
+    /// Garbage collection options for the compacted directory.
+    pub compacted_options: Option<GarbageCollecterDirectoryOptions>,
+
+    /// An optional tokio runtime handle to use for scheduling garbage collection. You can use
+    /// this to isolate garbage collection to a dedicated thread pool.
+    pub compaction_runtime: Option<Handle>,
+}
+
+impl Default for GarbageCollecterDirectoryOptions {
+    fn default() -> Self {
+        Self {
+            poll_interval: Duration::from_secs(60),
```

Comment:
Set to five minutes. I'm just envisioning people showing up in Discord shouting that the compactor isn't running because they haven't waited 5 minutes. 😅 Still, it's a better default.